src/constants.js

```diff
@@ -13,6 +13,11 @@ export const NAMESPACES = /** @type {const} */ ([
 /** @type {ReadonlyArray<Namespace>} */
 export const PRESYNC_NAMESPACES = ['auth', 'config', 'blobIndex']
 
+/** @type {ReadonlyArray<Namespace>} */
+export const DATA_NAMESPACES = NAMESPACES.filter(
+  (namespace) => !PRESYNC_NAMESPACES.includes(namespace)
+)
```

Comment:
This was removed recently in 2b36edd45f82729d5df841937bc915e8d5d3adc7, and I'm adding it back same as it was.